cmd/openshift-install/create.go

```diff
@@ -422,6 +423,10 @@ func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset
 				timeout = 60 * time.Minute
 			}
 		}
+
+		if agentConfig, err := assetStore.Load(&agentconfig.AgentConfig{}); err == nil && agentConfig != nil {
```

Comment:
I'd suggest to refactor these lines a in func, just to remove the duplication, and in case it will be needed in future by some other command

Reply:
Good idea. Updated.